draft-ietf-quic-ack-frequency.md

```diff
@@ -403,21 +403,33 @@ controllers.
 ## Congestion Control
 
 A sender needs to be responsive to notifications of congestion, such as
-a packet loss or an ECN CE marking. Also, window-based congestion controllers
-that strictly adhere to packet conservation, such as the one defined in
-{{QUIC-RECOVERY}}, rely on receipt of acknowledgments to send additional data into
-the network, and will suffer degraded performance if acknowledgments are delayed
-excessively.
-
-To enable a sender to respond to potential network congestion, a sender SHOULD
-cause a receiver to send an acknowledgement at least once per RTT if there are
-unacknowledged ack-eliciting packets in flight. A sender can accomplish this by
-sending an IMMEDIATE_ACK frame once per round-trip time (RTT), or it can set the
-Ack-Eliciting Threshold and Request Max Ack Delay values to be no more than a
-congestion window and an estimated RTT, respectively. If the sender is
-application-limited, a large ack delay can delay acknowledgement information
-unnecessarily when entering idle periods. Therefore, if no further data is buffered to be
-sent, a sender can send an IMMEDIATE_ACK frame with the last data
+a packet loss or an ECN CE marking. To enable a sender to respond to potential
+network congestion in a timely fashion, usually at least one acknowledgement
+per round trip is needed if there are unacknowledged ack-eliciting packets
+in flight. A sender can accomplish this by setting the Ack-Eliciting Threshold
+to a value no larger than the current congestion window or the Request Max Ack
+Delay value to no more than the estimated round trip. A sender can typically
```

Comment:
```suggestion
Delay value to no more than the estimated round trip. Alternatively, a sender can
```

Reply:
I also proposed this sentence based on the discussion. We don't really need it but, I think in this part of the draft a bit of a warning could still be good:

> However, the congestion window particularly but also the RTT are dynamic and therefore might require frequent updates if the selected value are close to this limit.

Reply:
That's true.  Do you think we need to explicitly say that?